src/careers/current-openings/opening.html

```diff
@@ -0,0 +1,129 @@
+                'heading': 'Share this job',
+                'hide_heading': false,
+                'show_linkedin': true,
+                'additional_classes':
+                    'share__horizontal share__align-right'
+            }) }}
+            </div>
+        </div>
+    </section>
+
+    <section class="block
+                block__padded-top
+                block__border-top">
+        <h2>Job Description</h2>
+        <p>Lorem ipsum...place description text here</p>
+
+        <h2>Duties</h2>
+        <h3>As a test opening, you will:</h3>
+        <ul class="list__branded">
+            <li>Lorem ipsum...place first duty here</li>
+            <li>Lorem ipsum..place second duty here</li>
+            <li>Etc, etc, etc</li>
+        </ul>
+
+        <h2>Departments</h2>
+        <p>Multiple offices</p>
+    </section>
+
+    <section class="block
+                block__padded-top
+                block__border-top">
+        <div class="content-l">
+            <div class="content-l_col content-l_col-1-2">
+                <h2 class="u-mb0">Interested in Applying?</h2>
+            </div>
+            <div class="content-l_col content-l_col-1-2">
+                {{ share.render({
+                    'title': 'Test Opening Title',
+                    'heading': 'Share this job',
+                    'hide_heading': false,
+                    'show_linkedin': true,
+                    'additional_classes': 'share__horizontal share__align-right'
+                }) }}
+            </div>
+        </div>
+        <div class="block
+                    block__bg
+                    block__border
+                    block__sub">
+            <h3>Before you apply</h3>
+            <p class="short-desc">
+                Lorem impsum...replace with application info paragraph
+            </p>
+
+            <ul class="list list__links">
+                <li class="list_item">
+                    <a class="jump-link" href="#">
+                        Learn about working @ CFPB
+                    </a>
+                </li>
+                <li class="list_item">
+                    <a class="jump-link" href="#">
+                        Learn about the application process
+                    </a>
+                </li>
+            </ul>
+        </div>
+        <div class="block block__flush-top">
+            <h4>Open to All Citizens (Excepted service - Permanent)</h4>
+            <p class="short-desc">Applications will be acceptes from U.S. citizens....</p>
+            <div class="content-l">
+                <div class="content-l_col content-l_col-1-4">
+                    <a class="btn" href="#">Apply now</a>
+                </div>
+                <div class="content-l_col content-l_col-3-4">
+                    <h4 class="u-mb0">You are about to leave consumerfinance.gov.</h4>
+                    <p class="short-desc">To finish the application, you must go to USAJobs.gov.</p>
```

Comment:
USAJobs.gov should not be a link. If it's a link, folks will click and expect to be taken to their application and then be :open_mouth: :question:  when they're on the homepage.